model/logging/Logger.php

```diff
@@ -136,6 +151,9 @@ private function log($level, $message, array $context = [])
         $message = $this->interpolateMessage($message, $context);
 
         $this->logToSyslog($level, $message, $context);
+        if ($this->isDbLogEnabled()) {
```

Comment:
It will be less readable I think. You need to go to the method `logToDatabase()` to check if it actually log to database.